### PR TITLE
adding gdpr support to BRT adapter

### PR DESCRIPTION
**Pull request by alexecookbrt**
let me know if any other steps are necessary to make BRT adapter officially GDPR supported in IX

### CODE REVIEW

File: b-real-time-htb.js [hunk 1]
```diff
@@ -154,28 +156,49 @@ function BRealTimeHtb(configs) {
...
+            });
+            tag.id = returnParcel.xSlotRef.placementId;
+            tag.primary_size = tag.sizes[0];
+            tag.allow_smaller_sizes = false;
+            tag.prebid = true;
+            tag.disable_psa = true;
+            tag.uuid = uuid;
+            returnParcel.uuid = uuid;
+
+            if (gdprPrivacyEnabled) {
```
Comment (ix-certification) [inline b-real-time-htb.js:177]: `gdprPrivacyEnabled` is undefined

**Comment by ix-certification** [comment]
Please define and add:

`var ComplianceService;`

In addition please add to the constructor:

`ComplianceService = SpaceCamp.services.ComplianceService;`

**Comment by alexecookbrt** [comment]
We've made these updates in our most recent commit. Let us know how it looks. Thanks!

**Comment by ix-certification** [comment]
Hi, 
The changes looks ok and here's the request being generated from the wrapper: 
```{"tags":[{"sizes":[{"width":970,"height":250},{"width":728,"height":90}],"id":10256447,"primary_size":{"width":970,"height":250},"allow_smaller_sizes":false,"prebid":true,"disable_psa":true,"uuid":"SpltXVIM","gdpr_consent":"test","gpdr":"1"}]}```

On the other hand, the response from your endpoint returns 
```{"error":"Could not find valid member in request"} ```

Which caused an error on line 
https://github.com/indexexchange/b-real-time-adapter/blob/master-v2/b-real-time-htb.js#L273, due to line 
https://github.com/indexexchange/b-real-time-adapter/blob/master-v2/b-real-time-htb.js#L258
as adResponse doesn't have `tags`.

Can you please add additional checks around adResponse to ensure when no valid bid exists a pass event similar to 
https://github.com/indexexchange/ix-base-adapter/blob/master-v2/ix-base-htb.js#L301-L307 
is triggered?

**Comment by alexecookbrt** [comment]
Thanks for the feedback, we've pushed an update to return if tags doesn't exist.

**Comment by ix-certification** [comment]
Hi,
These changes look good. Can you please confirm one final time that this request generated by the wrapper looks like you're expecting?

`{"tags":[{"sizes":[{"width":300,"height":250}],"id":654321,"primary_size":{"width":300,"height":250},"allow_smaller_sizes":false,"prebid":true,"disable_psa":true,"uuid":"fTNAIn4b","gdpr_consent":"BOPnGWyORXuTCABABAENBV-AAAAe97_______9______9uz_Gv_r_f__33e8_39v_h_7_-___m_-3zV4-_1vV11yPg1urfIr1NpjQ6OGgAAAAAAA","gpdr":"1"},{"sizes":[{"width":300,"height":250}],"id":54321,"primary_size":{"width":300,"height":250},"allow_smaller_sizes":false,"prebid":true,"disable_psa":true,"uuid":"cqyavpgg","gdpr_consent":"BOPnGWyORXuTCABABAENBV-AAAAe97_______9______9uz_Gv_r_f__33e8_39v_h_7_-___m_-3zV4-_1vV11yPg1urfIr1NpjQ6OGgAAAAAAA","gpdr":"1"},{"sizes":[{"width":300,"height":600}],"id":12345,"primary_size":{"width":300,"height":600},"allow_smaller_sizes":false,"prebid":true,"disable_psa":true,"uuid":"OFsq3rZ2","gdpr_consent":"BOPnGWyORXuTCABABAENBV-AAAAe97_______9______9uz_Gv_r_f__33e8_39v_h_7_-___m_-3zV4-_1vV11yPg1urfIr1NpjQ6OGgAAAAAAA","gpdr":"1"}]}`

In particular, we see the GDPR information being sent repeatedly, once for each slot in the request. Is this how your endpoint is expecting to receive the data?

**Comment by alexecookbrt** [comment]
Yes this looks fine, we will read gdpr on each tag and on the page to confirm. Thank you for looking into this.

**Pull request merged**